webui/react/src/hooks/useModal/Project/useModalProjectCreate.tsx

```diff
@@ -20,12 +21,22 @@ interface Props {
 const useModalProjectCreate = ({ onClose, workspaceId }: Props): ModalHooks => {
   const [ name, setName ] = useState('');
   const [ description, setDescription ] = useState('');
+  const [ isNameUnique, setIsNameUnique ] = useState<boolean>(true);
 
   const { modalOpen: openOrUpdate, modalRef, ...modalHook } = useModal({ onClose });
 
+  const findIsNameUnique = useCallback(async (name: string) => {
```

Comment:
**question:** does the API endpoint of creating the project return an error if a project with the same name has been provided? If so, it might be better to just let the creation of the project catch this problem instead of proactively trying to fetch the full list of project names each time and compare it to the input.

Reply:
Yes, API catches the error. I implemented cuz useModalModelCreate.tsx has one but not others. I'll remove it from all